swim/test_utils.go

```diff
@@ -146,6 +146,23 @@ func bootstrapNodes(t *testing.T, testNodes ...*testNode) {
 		})
 		require.NoError(t, err, "node must bootstrap successfully")
 	}
+
+	if waitForConvergence {
```

Comment:
Agree with @dansimau. We should verify an empty dissemination list and membership checksums.